### DISCUSSION TITLE
Microsoft planning to remove VB Script

### DISCUSSION THREAD

**Discussion by Fredaker** [PSAppDeployToolkit/PSAppDeployToolkit#841] [Q&A]
Hello,
Microsoft planning to remove VB Script.(https://learn.microsoft.com/en-us/windows/whats-new/deprecated-features-resources#vbscript)
Are you planning to update Execute-ProcessAsUser and Block-AppExecution functions to replace VBS?

**Comment by mjr4077au** [comment]
This has been addressed in PSADT v4.
